packages/backend-app-api/config.d.ts

```diff
@@ -33,6 +33,37 @@ export interface Config {
        */
       dangerouslyDisableDefaultAuthPolicy?: boolean;
 
+      /** Controls how to store keys for plugin-to-plugin auth */
+      keyStore?:
+        | { type: 'database' }
+        | {
+            type: 'static';
+            static: {
+              /**
+               * Must be declared at least once and the first one will be used for signing.
+               */
+              keys: Array<{
+                /**
+                 * Path to the public key file in the SPKI format. Should be an absolute path.
+                 */
+                publicKeyFile: string;
+                /**
+                 * Path to the matching private key file in the PKCS#8 format. Should be an absolute path.
+                 */
+                privateKeyFile: string;
```

Comment:
Should this perhaps be optional so that you don't need to include it for 2+ keys?

Reply:
yep, fixed